ckanext/validation/tests/test_plugin.py

```diff
@@ -22,7 +22,7 @@ def _assert_validation_enqueued(mock_enqueue, resource_id):
 class TestResourceControllerHooksUpdate(object):
 
     @change_config('ckanext.validation.run_on_create_async', False)
-    @mock.patch('ckantoolkit.enqueue_job')
```

Comment:
Why were these changed?

Reply:
It was changed, because I change `ckantoolkit` import to `ckan.plugins.toolkit`, but then I bring it back, because seems like using `ckantoolkit` is better for backward compatibility